deploy/ui/form.json

```diff
@@ -107,26 +107,64 @@
           ]
         },
         {
-          "label": "Admin user",
-          "type": "fieldGroup",
+          "label": "Admin user configuration",
+          "type": "dropDown",
           "required": false,
-          "jsonPath": "$.spec.commonConfig",
+          "jsonPath": "$.spec.commonConfig.AdminUser",
```

Comment:
is this json path correct?

Reply:
if it is, sould be adminUser instead AdminUser

Reply:
path correct but first letter lowercase